src/Symfony/Bridge/Twig/Mime/BodyRenderer.php

```diff
@@ -44,7 +45,12 @@ public function render(Message $message): void
             return;
         }
 
-        $vars = array_merge($this->context, $message->getContext(), [
+        $messageContext = $message->getContext();
+        if (isset($messageContext['email'])) {
+            throw new InvalidArgumentException(sprintf('A "%s" context cannot have an "email" entry as this is a reserved variable.', TemplatedEmail::class));
```

Comment:
```suggestion
            throw new InvalidArgumentException(sprintf('A "%s" context cannot have an "email" entry as this is a reserved keyword.', TemplatedEmail::class));
```

Reply:
Not sure about the change. The context is about the variables that will be accessible in the Twig template.